### ISSUE TITLE
Cannot Add Skill Specializatiojns

### ISSUE THREAD

**Issue by NerdEyeIndustries** [NickEastNL/fvtt-doctor-who#9]
GM or Player attempts to click on + in a skill, no love. no response from the UI..

**Comment by NerdEyeIndustries** [comment]
I also have the same problem with NPCs

**Comment by NickEastNL** [comment]
You are correct, this was an oversight. Unfortunately, I won't be able to fix it any time soon, but I _will_ fix it. At most it would have to wait until after the full release of Foundry V12. I can't promise anything else.

**Comment by NerdEyeIndustries** [comment]
Seriously? Wow, so much for using this any time soon
